applications/asset_tracker_v2/src/modules/Kconfig.modules_common

```diff
@@ -4,6 +4,20 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+menuconfig MODULES_COMMON
+	bool "Modules common"
+	default y
+	help
+	  Library containing functionality that is shared between all modules in the application.
+
+if MODULES_COMMON
+
+config MODULES_COMMON_MODULES_COUNT_MAX
+	int "Maximum number of modules in the application."
+	default 8
```

Comment:
Please explain `default 8` and why the maximum number of modules need to be configured.

Reply:
The implementation have changed. This Kconfig is no longer needed.